### PR TITLE
add argument for plotting just specific groups

### PR DESCRIPTION
**Pull request by stphnma**

### CODE REVIEW

File: convoys/plotting.py [hunk 1]
```diff
@@ -15,25 +15,38 @@
 
 
 def plot_cohorts(G, B, T, t_max=None, model='kaplan-meier',
-                 ci=None, plot_kwargs={}, plot_ci_kwargs={}, groups=None):
+                 ci=None, plot_kwargs={}, plot_ci_kwargs={}, groups=None, specific_groups=None):
```
[the remark this answers is not in the document]
Comment (erikbern) [reply convoys/plotting.py]: you probably want to make G, B, T optional here too if you supply a pre-trained model

**Comment by erikbern** [comment]
👍 if you can fix the hound warnings

i think the plotting interface actually makes a lot more sense to take a trained model as an argument. i'm fine removing the old interface (i can do in a subsequent PR)

File: convoys/plotting.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (stphnma) [reply convoys/plotting.py]: actually I think we still need it for these parts https://github.com/better/convoys/blob/abb6d2a807318e5c398d1a6347a9f44bd1919811/convoys/plotting.py#L37

[the remark this answers is not in the document]
Comment (erikbern) [reply convoys/plotting.py]: that's actually only needed for the legend anyway, might not be super important

File: convoys/plotting.py [hunk 2]
```diff
@@ -15,25 +15,42 @@
...
-        groups = set(G)
+        groups = list(set(G))
+    if model not in _models.keys():
+        if not isinstance(model, convoys.multi.MultiModel)):
+            raise Exception('model must be of %s or a convoys object' % str(_models.keys()))
 
-    # Fit model
-    m = _models[model](ci=bool(ci))
-    m.fit(G, B, T)
+    if not isinstance(model, convoys.multi.MultiModel):
```
[the remark this answers is not in the document]
Comment (stphnma) [reply convoys/plotting.py]: ? works fine when I run it

**Pull request merged**